compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -142,7 +142,7 @@ template <typename OpTy>
 static LogicalResult
 verifyGatherScatter(OpTy op, int64_t sliceRank, ShapedType originalType,
                     ShapedType updateType, StringRef originalName,
-                    StringRef updateName) {
+                    StringRef updateName, Region *region) {
```

Comment:
It looks easier if you refactor the region check to a new `verifyScatterRegion()` method.

Reply:
moved it to the scatter verifier only.